packages/Webkul/Product/src/Repositories/ProductCustomerGroupPriceRepository.php

```diff
@@ -22,6 +23,7 @@ public function model(): string
     public function saveCustomerGroupPrices(array $data, $product)
     {
         $previousCustomerGroupPriceIds = $product->customer_group_prices()->pluck('id');
+        $seenUniqueIds = [];
```

Comment:
Add a new line above and name should be more meaningfull like `processedUniqueIds`.